### ISSUE TITLE
AE2 REV3 version 10 + incompatibility.

### ISSUE THREAD

**Issue by mindcloud69** [TinyModularThings/IC2Classic#149]
ic2 Classic 1.1.3.3 and any ae2 rev2 ver 10 and higher. Crash on startup.

[Crash log gist](https://gist.github.com/mindcloud69/12e17c7b69d10af6c848)

**Comment by YuRaNnNzZZ** [comment]
Did you know how to read?
![](http://i.imgur.com/M9No5oU.png)
Latest stable version of AE2 (rv2-stable-10) is working with IC2 Classic without any problems.

**Comment by liach** [comment]
The new ae2 is too new for ic2 classic to support. I think you should wait for speiger.

**Comment by Speiger** [comment]
@mindcloud69 @liach  @YuRaNnNzZZ just delete the folder AEIntigration and everything that adds is removed (includes PTP busses and wrench support) And yeah i am thinking about to remove that thing again because there is no really use for it...

**Comment by Speiger** [comment]
I tested it with the dev version of AE2 (rv2-stable-10) and everything worked fine. Could be that they changed the stuff back but at least no crashes anymore. Soon i upload a fix to other stuff. Thanks

**Comment by mindcloud69** [comment]
Speiger, thank you for responding politely with a work-around. I understand the need to work on core problems first and I appreciate the work you are doing to make IC2 fun again. A suggestion, if I may, perhaps adding a config option to disable mod compatibility would be a good idea. I only opened this ticket after I determined that I could not disable the ae2 integration that was causing the crash.
